server/src/test/java/com/blackship/battlesheep/game/state/TestUtils.java

```diff
@@ -37,4 +38,15 @@ public static String generateBoardState(FieldState state) {
     public static List<Integer> generateEmptyList() {
         return new ArrayList<>();
     }
+
```

Comment:
You can use data providers.
https://examples.javacodegeeks.com/enterprise-java/testng/testng-dataprovider-example/